### PR TITLE
XW-1184 | Fix actions for RWA

### PR DESCRIPTION
**Pull request by vforge**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1184

## Reviewers

@bognix

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: front/main/app/utils/article-link.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ export default function getLinkInfo(basePath, title, hash, uri) {
 			};
 		}
 
-		if (!isMercuryNamespaceHandlingOverridden(article[3])) {
+		if (article[3] && !isMercuryNamespaceHandlingOverridden(article[3])) {
```
Comment (hakubo) [inline front/main/app/utils/article-link.js]: magic! what is article[3] ?

Comment (bognix) [inline front/main/app/utils/article-link.js]: there is some sort of regex above and I guess it's a matching group returned from this regex

Comment (hakubo) [inline front/main/app/utils/article-link.js]: any chance for a comment?

Comment (bognix) [inline front/main/app/utils/article-link.js]: @Warkot could you add a comment as part of XW-1104 work? this PR should be merge ASAP ?

Comment (vforge) [inline front/main/app/utils/article-link.js]: talked off-line, it will be added as a part of later ticket.

**Pull request merged**